swan/files/swan_config.py

```diff
@@ -24,9 +24,16 @@ def __init__(self, spawner, pod):
     def get_swan_user_pod(self):
 
         # pod labels
-        pod_labels = dict(
-            lcg_release = self.spawner.user_options[self.spawner.lcg_rel_field].split('/')[0]
-        )
+        if self.spawner.user_options[self.spawner.software_source] == self.spawner.lcg_special_type:
+            pod_labels = dict(
+                software_source = self.spawner.user_options[self.spawner.lcg_rel_field].split('/')[0]
```

Comment:
To be checked if the previous label here (lcg_release) was used somewhere else in our code.

Reply:
I think this label (lcg_release) was used for setting affinity with other pods that had the same LCG release some time ago:

https://github.com/swan-cern/swan-charts/commit/d8fb2dcdf9eab5e64cb5ccf78ea714d9facb9ecf

So it should not hurt to transform it into software_source. I can't find any occurrence of lcg_release at the moment anywhere else in the code.